### PR TITLE
Imp AA/SP registry

### PR DESCRIPTION
**Pull request by JeongGoEun**
Imp AA/SP registry

### CODE REVIEW

File: src/components/User.js [hunk 1]
```diff
@@ -1,10 +1,182 @@
...
-import { List } from 'antd';
+import { Table, Input, Modal, List } from 'antd';
+
+var structArr = [];
+var typeArr = ['Personal', 'Institution'];
+var titleArr = ['a', 'b', 'c', 'd', 'e', 'f', 'g'];
+var rollArr = ['Attestation Agency', 'Service Provider'];
+var metaidArr = ['0x1111111', '0x1111112','0x1111113','0x1111114', '0x1111115','0x1111116'];
+var registerdate;
+var user;
```
Comment (hexoul) [inline src/components/User.js]: where will it be used? never used.

File: src/components/User.js [hunk 2]
```diff
@@ -1,10 +1,182 @@
 import React from 'react';
-import { List } from 'antd';
+import { Table, Input, Modal, List } from 'antd';
+
+var structArr = [];
+var typeArr = ['Personal', 'Institution'];
+var titleArr = ['a', 'b', 'c', 'd', 'e', 'f', 'g'];
+var rollArr = ['Attestation Agency', 'Service Provider'];
+var metaidArr = ['0x1111111', '0x1111112','0x1111113','0x1111114', '0x1111115','0x1111116'];
```
Comment (hexoul) [inline src/components/User.js]: test with actual length of address, e.g., 0x7304f14b0909640acc4f6a192381091eb1f37701

File: src/components/User.js [hunk 3]
```diff
@@ -1,10 +1,182 @@
...
+var registerdate;
+var user;
+
+const Search = Input.Search;
+
+const columns = [{
```
Comment (hexoul) [inline src/components/User.js]: please fix code indent. If there is map in a list, more indent is needed

File: src/components/User.js [hunk 4]
```diff
@@ -1,10 +1,182 @@
...
+    });
+  }
+
+  componentWillMount() {
+    this.initialize();
+  }
+
+  initialize() {
+    var i;
+    for(i=0; i<20; i++) {
```
Comment (hexoul) [inline src/components/User.js]: coding convention. `for(` => `for (`

File: src/components/User.js [hunk 5]
```diff
@@ -1,10 +1,182 @@
...
+        metaID: metaidArr[Math.floor(Math.random() * 6)],
+        registerDate: Date.now() - Math.floor((Math.random()*10)),
+      });
+    }
+    this.setState({data: structArr});
+  }
+
+  onSearch(value) {
+    console.log('onSearch value: ',value);
+    if(value == '' || value == undefined) {
```
Comment (hexoul) [inline src/components/User.js]: coding convention. `if(` => `if (`

File: src/components/User.js [hunk 6]
```diff
@@ -1,10 +1,182 @@
...
+          style={{ width: 500, float: 'right', marginBottom: '20px' }}
+        />
+        <br />
+        <Modal
+          title={this.state.title}
+          visible={this.state.modalVisible}
+          onOk={this.handleOk}
+          onCancel={this.handleCancel}>
+          <div>
+            <h5 style={{ margin: '10px 0', float: 'right' }}>Registered on: {this.state.registerDate}</h5>
```
Comment (hexoul) [inline src/components/User.js]: h5 and h3 should be checked when it is normal on mobile view

File: src/components/User.js (hunk 1 above)
Comment (JeongGoEun) [reply src/components/User.js]: It is used when random information is inserted randomly for List.

File: src/components/User.js (hunk 6 above)
Comment (JeongGoEun) [reply src/components/User.js]: [](https://docs.usecreator.com/docs/headings)
Check it and confirm me

File: src/components/User.js (hunk 1 above)
Comment (rubis-lab) [reply src/components/User.js]: I meant `user` line 10

Comment (hexoul) [reply src/components/User.js]: I meant `user`, line 10

File: src/components/User.js (hunk 6 above)
Comment (hexoul) [reply src/components/User.js]: Why don't you use chrome developer tool?

**Pull request closed without merging**